src/main/java/com/mig82/folders/steps/FolderPropertiesStep.java

```diff
@@ -0,0 +1,92 @@
+package com.mig82.folders.steps;
+
+import com.mig82.folders.properties.FolderPropertiesUtil;
+import hudson.EnvVars;
+import hudson.Extension;
+import hudson.model.Job;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import org.jenkinsci.plugins.workflow.steps.*;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
```

Comment:
Missing Javadoc. It should state a description of what the class does and why. And optionally (if you wish) name you as author for future Q&A.